backend/elasticsearch/src/main/java/org/hibernate/search/backend/elasticsearch/orchestration/impl/ElasticsearchDefaultWorkBulker.java

```diff
@@ -60,6 +62,17 @@ public ElasticsearchDefaultWorkBulker(ElasticsearchWorkSequenceBuilder sequenceB
 
 	@Override
 	public <T> CompletableFuture<T> add(BulkableElasticsearchWork<T> work) {
+		DocumentRefreshStrategy workRefreshStrategy = work.getRefreshStrategy();
+		if ( currentBulkItems.isEmpty() ) {
+			currentBulkRefreshStrategy = workRefreshStrategy;
+		}
+		else if ( currentBulkRefreshStrategy != workRefreshStrategy ) {
+			// This work needs a bulk with a different "refresh" parameter; we can't reuse the current bulk.
+			addWorksToSequence();
+			finalizeBulkWork();
```

Comment:
It's not possible, because `finalizeBulkWork()` will end the current bulk and thus clear `currentBulkItems`. And `maxBulkSize` is always a large number: setting it to `1` would not make sense (what's a bulk of one element?).